### DISCUSSION TITLE
Could someone share a working DAB-Image ?

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1818] [Feature Requests]
I'm trying to get dab working , but I've no luck until now . Search is working , but it runs only 10 till 20 seconds without any sound .
Receiving 74 stations , so no probs with the antenna . I try all things from this forum to get it working .
I use a RPI 4  with a 4tb SSD and an usb dab-stick . Both are getting extra power .

**Comment by kan-ibal** [comment]
Ttry this file:
[dab-rtlsdr-3](https://github.com/kan-ibal/raudio-dab-lite/raw/refs/heads/main/dab-rtlsdr-3)
You  have to replace `/usr/bin/dab-rtlsdr-3` file. This is my compilation of dab-cmdline-0.8.
To decrease CPU usage you can change ffmpeg encoding in `/srv/http/bash/dab-start.sh` from 
```
-b:a 160k \
-c:a aac \
```
to
```
-b:a 128k \
-c:a mp2 \
```

or 
`curl -sL https://raw.githubusercontent.com/kan-ibal/raudio-dab-lite/refs/heads/main/install.sh | bash`
